botocore/docs/method.py

```diff
@@ -207,6 +208,17 @@ def document_model_driven_method(section, method_name, operation_model,
                 request_params_section, operation_model.input_shape,
                 include=include_input, exclude=exclude_input)
 
+
+    # Find idempotent params and append documentation
+    if 'request-params' in section.available_sections:
+        param_section = section.get_section('request-params')
+        for param_name in param_section.available_sections:
+            if param_name in context.get('special_shape_types', {}).get('idempotent_members', []):
+                docs = param_section.get_section(param_name).get_section('param-documentation')
+                docs.writeln('This field is autopopulated if not provided.')
```

Comment:
So based on the example you had. It looks like a block quote, which does not look good. Could remove ``writeln`` and use ``write`` instead to append on the value?